.gitignore

```diff
@@ -32,3 +32,5 @@ src/svtk/svtk/utils/utils.pyc
 src/svtk/svtk/vcfcluster.pyc
 /test_inputs
 /inputs/
+/wdl_test/.runs.json
+/wdl_test/.configs.json
```

Comment:
I was thinking of tracking this file via git. Mainly because once a test is created, it can be re-run using anyone with access to the carrot server. This would avoid redefining tests (and all their dependencies) in the carrot server.

Reply:
So thinking out loud here to make sure I understand, this file seems to define pipelines, templates, tests, and results (in the CARROT vernacular). So these are static metadata - they include entries like time stamps, a creator id, uuids, description strings, checksums, and element ids like `"name": "gatk-sv_1629914186"`. So some things are dependent on when/where the test was first created (time stamp, creator id), some on the version of code (checksus, uuids?), and is the suffix on `name` random or some kind of hash? 

Which of these metadata are used CARROT? If two developers generated this metadata file from scratch, the templates and results would have their own creator ids, timestamps, and `name` id? Would that prevent one from comparing their tests using the CARROT framework because they would be registered as different templates on the server?

Reply:
Good point, thanks. Actually, I guess I'll update the README to elaborate on this. 

A carrot "test" is defined based on test and evaluation WDLs and their inputs, then the test can be run multiple times. If the test or evaluation WDL or any of their inputs is changed, their related carrot resources need to be updated, otherwise, the created resources can be reused by any user who has access to that carrot server. The `CarrotHelper` automatically detects the changed resources and updates them as needed. Hence, if you just want to run the tests, you do not need to redefine them, and if you're modifying any of the tests, the related resources are updated automatically. Though there is one caveat; changes to the test and evaluation WDLs are detected only if they are committed and pushed to GitHub because carrot fetches these files from Github, hence, if you modify it locally and update the tests, carrot will still be using the older commit. This will potentially pollute your branch, but maybe a rebase can fix that.